docs/book/user-guide/advanced-guide/client.md

```diff
@@ -0,0 +1,180 @@
+- **Service Connectors**: The service connectors that that you have set up to
+[connect ZenML to your infrastructure](../../platform-guide/set-up-your-mlops-platform/connect-zenml-to-infrastructure.md).
+
+## Client Methods
+
+### CRUD (Create, Read, Update, Delete)
+
+For most resources, the ZenML Client provides five different CRUD methods:
+
+#### List
+
+Get a list of resources, e.g.:
+
+```python
+client.list_pipeline_runs(
+    stack_id=client.active_stack_model.id,  # filter by stack
+    user_id=client.active_user.id,  # filter by user
+    sort_by="desc:start_time",  # sort by start time descending
+    size=10,  # limit page size to 10
+)
+```
+
+These methods always return a 
+[Page](https://sdkdocs.zenml.io/0.41.0/core_code_docs/core-models/#zenml.models.page_model)
+of resources that contains, by default, the first 50 results. You can modify the
+page size by passing the `size` argument or fetch a subsequent page by passing 
+the `page` argument to the list method.
+
+You can further restrict your search by passing additional arguments that will
+be used to filter the results. E.g., most resources have a `user_id` associated
+with them that can be set to only list resources created by that specific user. 
+The available filter argument options are different for each list method, check 
+out the method declaration in the
+[Client SDK documentation](https://sdkdocs.zenml.io/latest/core_code_docs/core-client/)
+to find out which exact arguments are supported or have a look at the fields
+the corresponding [filter model class](#resource-models).
+
+Except for pipeline runs, all other resources will by default be ordered by
+creation time ascending. E.g., `client.list_artifacts()` would return the
+first 50 artifacts ever created. You can change the ordering by specifying the
+`sort_by` argument when calling list methods.
+
+#### Get
+
+Fetch a specific instance of a resource by either resource ID, name, or name 
+prefix, e.g.:
+
+```python
+client.get_pipeline_run("413cfb42-a52c-4bf1-a2fd-78af2f7f0101")  # ID
+client.get_pipeline_run("first_pipeline-2023_06_20-16_20_13_274466")  # Name
+client.get_pipeline_run("first_pipeline-2023_06_20-16")  # Name prefix
+```
+
+#### Create, Update, Delete
+
+Methods for creating / updating / deleting resources are only available for some
+of the resources and the required arguments are different for each resource.
+Checkout the [Client SDK Documentation](https://sdkdocs.zenml.io/latest/core_code_docs/core-client/)
+to find out whether a specific resource supports write operations through the
+Client and which arguments are required.
+
+### Active User and Stack
+
+For some use cases you might need to know information about the user that you
+are authenticated as or the stack that you have currently set as active. You
+can fetch this information via the `Client.active_user` and 
+`Client.active_stack_model` properties respectively, e.g.:
```

Comment:
Maybe here is the place for a small educational sidebar / info foldout on the use of the word 'model' and how it derives from Pydantic and doesn't relate to ML models?